### ISSUE TITLE
Copy of BUG: When user try to sign up with existing email account, wrong error is thrown

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#6838]
**Describe the bug**

**To Reproduce**

1. Go to acc-alkem.io
2. Register new user with email that exists on the platform
3. Check the error: `You're trying to sign in with a LinkedIn/Microsoft account, but the associated email address was already used before. Please sign in with this e-mail address and password or try to reset the password. If that doesn't work, please send an email to support@alkem.io.`
<img width="1573" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNzRnQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8da89cee48436a8dd97665d6558bc41f1ba79131/image.png" alt="image.png" />

**Expected behavior**
Error must mention, that there is already user registered with such email.

**Comment by valentinyanakiev** [comment]
I am not sure why this is using the bug template, given there was already a story about it in the past and it was never supported natively by Ory Kratos. `Session` hook is not supported after verification (still unsupported in v1.2). We can try using a [web hook ](https://www.ory.sh/docs/guides/integrate-with-ory-cloud-through-webhooks)and work it around. Web hook is supported in the schema, e.g.:

```json
    verification:
      enabled: true
      ui_url: http://localhost:3000/verify
      after:
        default_browser_return_url: http://localhost:3000/login
        hooks:
          - hook: web_hook
            config:
              url: https://your-service.example.com/verification-complete
              method: POST
              body:
```

It will be some custom work thou, it is not coming out-of-the-box.

**Comment by techsmyth** [comment]
@valentinyanakiev agreed not a bug, it was that I copied another issue + the label came with it. Might be still worth doing this work as it is a real disconnect in the login flow at the moment, and taking away even one step would be a big deal. @SimoneZaza
